public/components/custom_panels/helpers/utils.tsx

```diff
@@ -384,6 +391,183 @@ export const renderCatalogVisualization = async ({
   setIsLoading(false);
 };
 
+const createOtelVisualizationMetaData = (
+  documentName: string,
+  visualizationType: string,
+  startTime: string,
+  endTime: string,
+  queryData: object
+) => {
+  return {
+    name: documentName,
+    description: '',
+    query: '',
+    type: visualizationType,
+    subType: PPL_METRIC_SUBTYPE,
+    metricType: OTEL_METRIC_SUBTYPE,
+    selected_date_range: {
+      start: startTime,
+      end: endTime,
+      text: '',
+    },
+    selected_timestamp: {
+      name: 'timestamp',
+      type: 'timestamp',
```

Comment:
Does Otel use `timestamp` as timestamp field name?

Reply:
Do otel fields not work with timerange filter/span filter? Can I save an OTEL metric and view it in event explorer. If yes, then what time field is used for date range filters?

Reply:
currently otel metrics is not supported in event analytics for P0